examples/checker/main.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Checker is a troubleshooting and auto-remediation tool to help you identify
+// issues that won't let your system boot.
+// It will simply run through a checklist of checks and optional remediations,
+// and continue until it either succeeds or fails without recovery.
+//
+// The checklist is specified via configuration files in JSON format. Run the
+// `genconfig` subcommand to get an example configuration file.
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"html/template"
```

Comment:
Why `html/template` (not `text/template`?)

Reply:
I think `goimports` messed up - the intention was to import `text/template`. Good catch, this should also shave a few tens of kbs :)

Reply:
well, ~100kb compressed, which is not too bad but probably still not good enough :(

Reply:
Try to compare `go tool nm -size -sort size /path/to/binary` outputs between master and your branch.